README.md

```diff
@@ -497,16 +506,7 @@ This sets the Service information fields as follows:
 
 #### edgex_profile
 
-This environment variable overrides the command line `profile` argument. It will replace the current value passed via the `-p` or `--profile`, if one exists. If not specified it will add the `--profile` argument. This is useful when running the service via snaps or docker compose.
-
-Using snaps:
-
-```
-sudo snap set app-service-configurable edgex_profile=http-export
-sudo snap start --enable edgex-app-service-configurable.app-service-configurable
-```
-
-This sets the `--profile=http-export` command line argument so that the application service uses the `http-export` configuration profile which resides at  `/res/http-export/configuration.toml`
+This environment variable overrides the command line `profile` argument. It will replace the current value passed via the `-p` or `--profile`, if one exists. If not specified it will add the `--profile` argument. This is useful when running the service docker compose.
```

Comment:
```suggestion
This environment variable overrides the command line `profile` argument. It will replace the current value passed via the `-p` or `--profile`, if one exists. If not specified it will add the `--profile` argument. This is useful when running the service via docker compose.
```

Also the title of this section is profile, should it be also tied to environment overrides?  Something like `profile through environment overrides`

Reply:
It is already a subsection of `Environment Variable Overrides` and the title is `edgex_profile` which is the name of the env variable.

Reply:
got cha